crates/apps_lib/src/cli.rs

```diff
@@ -5236,6 +5265,29 @@ pub mod args {
         }
     }
 
+    impl CliToSdk<Complete> for Complete {
+        type Error = std::io::Error;
+
+        fn to_sdk(self, _ctx: &mut Context) -> Result<Complete, Self::Error> {
+            Ok(Complete { shell: self.shell })
+        }
+    }
+
+    impl Args for Complete {
+        fn parse(matches: &ArgMatches) -> Self {
+            let shell = SHELL.parse(matches);
+            Self { shell }
+        }
+
+        fn def(app: App) -> App {
+            app.arg(
+                SHELL.def().help(wrap!(
+                    "Which shell to generate the completions for."
```

Comment:
nit: this is more consistent with the rest of the help messages
```suggestion
                    "The shell to generate the completions for."
```